lib/stdlib/src/math.erl

```diff
@@ -134,8 +134,31 @@ tan(_) ->
 tanh(_) ->
     erlang:nif_error(undef).
 
+-spec ceilf(X) -> float() when
+      X :: number().
+ceilf(_) ->
+    erlang:nif_error(undef).
+
+-spec floorf(X) -> float() when
+      X :: number().
+floorf(_) ->
+    erlang:nif_error(undef).
+
 %%% End of BIFs
 
+-spec ceil(X) -> integer() when
```

Comment:
Yes, good point.  I modified the test to avoid the mistake with the constant and do get slower results with the if.  I didn't realize '+' is also a bif and that the if statement adds more work in addition to 2 bif calls.  The bignum handling is the only other concern, though that should only matter for the float ceil/floor taking integers.  It would be nice to finally have a ceil/floor function in Erlang, just since it has been a common question with many different implementation results of various degrees of quality.